File: s05_OOP/OOPProject/app/src/main/java/com/omeryildizce/oopproject/MainActivity.java
package com.omeryildizce.oopproject;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        User myUser = new User();
        myUser.name = "Ömer";
        myUser.job = "Unemployed";

        User newUser = new User();
        newUser.name = "Ali";
        newUser.job = "None";

        

    }
}

File: s15_KotlinLearning/Simpson/app/src/main/java/com/omeryildizce/simpson/MainActivity.kt
package com.omeryildizce.simpson

import android.graphics.Color
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.omeryildizce.simpson.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    private lateinit var name: String
    private lateinit var job: String
    private var age: Int = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)


    }

    fun makeSimpson(view: View) {
        name = binding.editTextSimpsonName.text.toString()
        job = binding.editTextTextSimpsonJob.text.toString()
        age = binding.editTextSimpsonAge.text.toString().toInt()
        val simpson = Simpsons(name, job, age)
        simpson.changeHairColor()
        binding.textViewSimpsonInformation.setText(simpson.toString())


    }
}